azure-pipelines.release-fluentui.yml

```diff
@@ -1,74 +1,220 @@
 pr: none
 trigger: none
 
-# Customize build number to include major version
-# Example: fluentui_20190626.1
-name: 'fluentui_$(Date:yyyyMMdd)$(Rev:.r)'
+schedules:
+  - cron: 0 3 * * *
```

Comment:
For those who aren't fluent in Unix, it would be helpful to have a comment about what this means

Reply:
Oh nice catch, I didn't realize this is in UTC until I saw azure-pipeline.release.yml